gender_analysis/analysis/gender_adjective.py

```diff
@@ -367,3 +367,13 @@ def get_top_adj(full_results, num):
         top[gender] = sorted(excluded_results[gender].items(), reverse=True, key=lambda x:x[1])[0:num]
 
     return top
+
+def display_gender_adjectives(result_dict,num_to_return):
+    '''takes the results of find_gender_adj and prints out to the user the top number_to_return
```

Comment:
As a general rule, we're trying to avoid statements like `print` if the function isn't doing any printing. Could you maybe switch this to some different phrasing that explains that it just returns the top?

Reply:
Hi Sam! Thanks so much for the comments on styling! I think I could just move the function to a new branch and fix the styling issues! Ah, looks like it is a PR for the whole branch and we certainly don't want that!

Thanks!